controller/monitor/disk_monitor.go

```diff
@@ -278,14 +280,14 @@ func (m *NodeMonitor) collectDiskData(node *longhorn.Node) map[string]*Collected
 			continue
 		}
 
-		orphanedReplicaDirectoryNames, err = m.getOrphanedReplicaInstanceNames(disk.Type, node, diskName, diskConfig.DiskUUID, disk.Path, replicaInstanceNames)
+		orphanedReplicaInstanceNames, err = m.getOrphanedReplicaInstanceNames(disk.Type, node, diskName, diskConfig.DiskUUID, disk.Path, replicaInstanceNames)
```

Comment:
This looks like it still returns orphaned replica directory names. Why the change in the variable name?

Reply:
It returnn replica lvol bdev for v2 volunes, so using orphanedReplicaInstanceNames is better.